src/main/java/com/opentok/OpenTok.java

```diff
@@ -134,7 +134,7 @@ public String generateToken(String sessionId, TokenOptions tokenOptions) throws
         } catch (UnsupportedEncodingException e) {
             throw new InvalidArgumentException("Session ID was not valid");
         }
-        if (!sessionIdParts.contains(Integer.toString(this.apiKey))) {
```

Comment:
Was removing `this` intentional? While it's probably going to scope correctly, it does clutter the PR.